src/projetos/CadastroProjetosPage.tsx

```
@@ -1,395 +1,527 @@
+        if (projeto) {
+          form.reset({
+            id: projeto.id ?? undefined,
+            titulo: projeto.titulo,
+            descricao: projeto.descricao,
+            publicoAlvo: projeto.publicoAlvo,
+            justificativa: projeto.justificativa,
+            dataInicio: projeto.dataInicio,
+            dataFim: projeto.dataFim,
+            restricaoPublico: projeto.restricaoPublico,
+            vinculadoDisciplina: projeto.vinculadoDisciplina,
+            equipeExecutora: projeto.equipeExecutora?.map((p: any) => p.nome) || [],
```

ESLint

[error] Insert `⏎·············`

(prettier/prettier)